assets/src/edit-story/components/colorPicker/colorPicker.js

```diff
@@ -165,7 +165,7 @@ function ColorPicker({
             onChange={updateCurrentColor}
             showOpacity={hasOpacity}
           />
-          {renderFooter && renderFooter(generatedColor)}
+          {renderFooter && renderFooter(color)}
```

Comment:
I'm not sure I understand how it partially worked before and fully works now? What's the essence of this change?

Reply:
It looked like `generatedColor` was always `null` when the color picker rendered unless making any changes: `null` resulting in the preset action not working.

At least that's what it seemed that the issue was -- after you asked, I tested it more, and it looks like each change in the color picker causes quite a few re-renders, `console.log` output for `generatedColor` (outside of effects) is this:
<img width="548" alt="Screenshot 2020-05-21 at 19 08 48" src="https://user-images.githubusercontent.com/3294597/82585826-f53c7080-9b96-11ea-8851-130240228d0d.png">
 
==> Result: Adding to color preset action will always receive `null` value by the time of clicking.

I'm not super familiar with the inner life of `generatedColor` vs `color` vs `currentColor` -- does it seem like something is not working as expected with the `colorPicker` and needs further investigation?

(I can look more into this tomorrow)

Reply:
Well, if it works now, it works.